EatGoodNaija.Server/Controllers/CartController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using EatGoodNaija.Server.Model;
+using EatGoodNaija.Server.Services.Interface;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EatGoodNaija.Server.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CartController : ControllerBase
+    {
+        private readonly IDishService _dishService;
+
+        public CartController(IDishService dishService)
+        {
+            _dishService = dishService;
+        }
+
+        [HttpPost("addtocart")]
+        public async Task<ActionResult<CustomerCart>> AddToCart(CustomerCart cart)
+        {
+            try
+            {
```

Comment:
Your controller should not contain much code. E.g

 var response = await _dishService.GetCartDishAsync(id);
 return Ok(response);

The ActionResult will return appropriate response